packages/api/config/distribution.yml

```diff
@@ -2,7 +2,7 @@
 ApiDistribution:
   handler: index.distributionAppHandler
   timeout: 20
-  memory: 512
+  memory: '{{parent.api_lambda_memory}}'
```

Comment:
Did we just miss this before?

Reply:
I don't know if it was missed, or purposely left at 512, but all of my responses were taking >512 max memory (I think 518MB). So I bumped it up.

Reply:
Hmm, does that mean other people would have to configure their deployments with `api_lambda_memory` in order for this to not crash? Just seems like something that could be easily missed. 

~~It's also curious why the distribution API would exceed memory limits because it's not doing anything very complex.~~  Nevermind, this is probably just because the endpoint is running Express. And I see we already do the same thing for the `ApiEndpoints` lambda, so this is fine